www/src/components/repository/RepositoryDescriptionMarkdown.js

```diff
@@ -35,7 +35,7 @@ function getLastStringChild(children) {
   return lastChild
 }
 
-function MdPre({ children, ...props }) {
+export function MdPre({ children, ...props }) {
```

Comment:
could we move this to a place where it's a bit clearer that it's reusable (eg the utils directory?)

Reply:
I've switched to another component already and will remove that export tomorrow.